tools/generate-llms-files.py

```
@@ -0,0 +1,336 @@
+    if args.update:
+        # Generate and update both outputs in the root directory
+        print(f"🔄 Parsing {args.input} and updating outputs...", file=sys.stderr)
+
+        # 1. XML output to llms.xml (or llms-ctx.xml as appropriate)
+        xml_path = args.xml_out if args.xml_out else "llms.xml"
+        xml_content = generate_xml_context(parsed, args.base_dir)
+        try:
+            with open(xml_path, "w", encoding="utf-8") as f:
+                f.write(xml_content)
+            print(f"✅ Generated XML context: {xml_path}", file=sys.stderr)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)